config/initializers/assets.rb

```diff
@@ -9,4 +9,7 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += ["authentication.css"]
+Rails.application.config.assets.precompile += [
+  "authentication.css",
+  "cloud/bootstrap.js"
```

Comment:
Why not `//= require cloud/bootstrap` in application.js?

Reply:
Oh, because you are using it as an independent one in that specific page. Gotcha!

Reply:
Didn't see the need to load it on other pages, and have to add code to not error out on other pages ;-)